### PR TITLE
Desktop Edge 2.2.2.0 release notes

### PR DESCRIPTION
**Pull request by usha-mandya**
### Proposed changes

Release notes for Docker Desktop Edge 2.2.2.0

### Unreleased project version (optional)

2.2.2.0

### CODE REVIEW

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -12,6 +12,21 @@ For information about Stable releases, see the [Stable release
 notes](release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.2.0
```
Comment (nebuk89) [inline docker-for-mac/edge-release-notes.md]: Should this say Edge?

File: docker-for-windows/edge-release-notes.md [hunk 2]
```diff
@@ -12,6 +12,27 @@ For information about Stable releases, see the [Stable release
 notes](release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.2.0
```
Comment (nebuk89) [inline docker-for-windows/edge-release-notes.md]: Should this also say edge? :D

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (mat007) [reply docker-for-mac/edge-release-notes.md]: This has always been Community apparently, they are different pages all together for Stable and Edge.

Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md]: Yes, we have specific pages for Stable and Edge release notes and both pages state they are the 'Community' releases.

**Pull request merged**